src/allegro_flare/attributes.cpp

```diff
@@ -569,7 +569,7 @@ namespace allegro_flare
 
    int Attributes::__find_attribute_index(std::string key)
    {
-      for (int i=0; i<attributes.size(); i++)
+      for (long unsigned int i=0; i<attributes.size(); i++)
```

Comment:
Would this work as simply `unsigned`?

Reply:
Yeah, actually it would... I change it to `long unsigned int` because `attributes.size()` type was `long unsigned int`.